### PR TITLE
Release 24.2

### PR DESCRIPTION
**Pull request by Deflaimun**
## Description

Release 24.2

### CODE REVIEW

File: antora.yml [hunk 1]
```diff
@@ -11,8 +13,8 @@ asciidoc:
     # Fallback versions
     # We try to fetch the latest from GitHub at build time
     # --
-    full-version: 24.1.1
-    redpanda-beta-version: 24.1.7-rc1
+    full-version: 24.2.1
+    redpanda-beta-version: 24.2.1-rc5
```
Comment (Feediver1) [inline antora.yml]: In tonight's release sync meeting, @piyushredpanda mentioned that the build would be rc7. Please doublecheck.

Comment (piyushredpanda) [reply antora.yml]: I am not sure what this represents, actually. Our [beta] was v24.2.1-rc1...and all the intervening RCs until now have been ... what, release-candidates?

But yes the GA tag is going to be same as v24.2.1-rc7 tag.

Comment (Deflaimun) [reply antora.yml]: This is not ready yet. Please don't consider

Comment (Deflaimun) [reply antora.yml]: Here's the actual values 
https://github.com/redpanda-data/docs/pull/653

**Review by Feediver1 — APPROVED** [review]

**Pull request merged**